src/index.ts

```diff
@@ -102,69 +102,63 @@ function _hydrate(
     }
   }
 
-  if (line.replacements.length > 0) {
```

Comment:
this looks like a lot of diffs but basically just simplified the logic by removing the "if..." since the for loop already only runs if there are entries and then the end logic is the same of appending the line text